pkg/sql/logictest/testdata/logic_test/udf_unsupported

```diff
@@ -1,74 +1,3 @@
-subtest disallow_udf_in_table
```

Comment:
TODO - ask for SQL review

I think this is what we want(?). But it'd be great for someone from SQL confirm